packages/v4/patternfly-docs/content/design-guidelines/components/label/label.md

```diff
@@ -5,6 +5,21 @@ section: components
 
 Use a **label** when you want to highlight an element on a page to draw attention to it or make it more searchable. Labels can also be used to tag items of the same category. If you want to show a count, use a [badge](https://www.patternfly.org/v4/components/badge) instead.
 
+
+
+## Elements
+You can further customize labels by making them filled or unfilled, dismissable, adding icons, or making them clickable links. You can use any combination of these elements in a single label.
+
+
+<img src="./img/label-features.png" alt="Examples of different label features" width="180"/>
+
+
+1. **Fill/Unfilled labels:** Offer customization for emphasis.
+2. **Dismissible label:** Can be dismissed by clicking on the “x”.
+3. **Label with icon:** Can be used to further clarify the label.
+4. **Clickable labels:** Has a hover state to show interactiveness.
+5. **Editable labels:** Has a dashed underline to communicate editability.
+
 ## Usage
```

Comment:
Hmm the content under this usage section feels like it should be under the "Variations"... but not sure how to go about re-organizing that. Feels like the usage section should focus on the usage of them (ie: when to use a label vs when not to) 

The variations sections makes complete sense as is though

Reply:
If you don't feel like tackling this one in this PR btw, feel free to just made a different issue to address this